app/content/serializers/strike.py

```diff
@@ -12,39 +14,27 @@ class Meta:
 
 
 class StrikeSerializer(BaseStrikeSerializer):
+    creator = DefaultUserSerializer(read_only=True)
+    event = EventListSerializer(read_only=True)
+    user = DefaultUserSerializer(read_only=True)
 
-    user = serializers.SerializerMethodField()
-    event = serializers.SerializerMethodField()
-    creator = serializers.SerializerMethodField()
+    class Meta:
+        model = Strike
+        fields = BaseStrikeSerializer.Meta.fields + ("user", "creator", "event")
+
+
+class UserInfoStrikeSerializer(StrikeSerializer):
+    user = None
+
+    class Meta:
+        model = Strike
+        exclude = ("user",)
+
+
+class RegistrationStrikeSerializer(StrikeSerializer):
```

Comment:
Det er lettere hvis du bygger opp hierarkiet motsatt, så du inkluderer felter i stedet for å ekskludere :)